scripts/locales/en.lua

```diff
@@ -714,6 +718,11 @@ local  en = {
    },
 
    if_stats_overview = {
+      snmp = "SNMP",
+      iface_switch_active_msg_pre = "The selected interface",
```

Comment:
@emanuele-f Thank you for pointing that out. I'll correct this kind of messages later on.